examples/flask/app.py

```diff
@@ -0,0 +1,125 @@
+from flask import Flask, request
+from werkzeug.exceptions import HTTPException
+from flask_sqlalchemy import SQLAlchemy
+
+PET_CATEGORIES = ['cat', 'dog']
+PET_REQUIRED_FIELD_ERROR = 'The name and category field is required.'
+PET_NAME_LENGTH_ERROR = 'The length of the name must be between 0 and 10.'
+PET_CATEGORY_ERROR = 'The category must be one of: dog, cat.'
```

Comment:
Maybe you want to generate the categories (behind the colon) from `PET_CATEGORIES`? But anyway, this is not production code, this is an example code :-)

Reply:
Some tests are fixed with the preset category of the fake data, they will fail if we use a randomly generated category. I moved the fake data into the `init_database` function to make it more clear and compact.